test_battleship.py

```diff
@@ -2,6 +2,7 @@
 import unittest
 from BattleShip import BattleShip
 from unittest.mock import patch
+from random import *
```

Comment:
generally good practice not to use `import *` as it imports more dependencies than what's needed. 
what specifically from the random module do you need?

Reply:
Resolved. I need randint function from the module, but I am not using it yet so I removed it.